packages/numpy/CONFIG

```diff
@@ -40,7 +44,7 @@ PYTHON_BUILD_OPTS += --fcompiler=None
 # is simply to permit this.
 
 build:
-	chmod +w $(srcdir)        # Ouch.
+	chmod -R +w $(srcdir)        # Ouch.
```

Comment:
Damn.  Could you identify which files are being added and overwritten during the build, please: maybe just drop them into a comment in the conversation.  I'd probably like to create an issue to remove this line altogether if possible...

Reply:
It's currently failing on cythonizing files (the error I get is for 'bounded_integers.pxd' but I suspect there are more files it's trying to cythonize into src).

Reply:
Ta.  Can you put this comment over in issue https://github.com/Araneidae/rootfs/issues/9, and we'll forget about it here.